### ISSUE TITLE
Increment and decrement operators behaviour.

### ISSUE THREAD

**Issue by xeioex** [nginx/njs#145]
```js
> --undefined
ReferenceError: Invalid left-hand side in prefix operation in shell:1
```
and

```js
> ++NaN
ReferenceError: Invalid left-hand side in prefix operation in shell:1
```

should return NaN instead.

**Comment by xeioex** [comment]
Will be automatically fixed after #132.

**Comment by crasyangel** [comment]
@xeioex @nginx-hg-mirror should closed?
